docs/tutorials/4. Developer notes.md

```diff
@@ -0,0 +1,18 @@
+
+# Developer notes
+
+## Good practices
+<!-- We recommend you to write your feature extraction process inside a WebWorker or as AudioWorklets. -->
+- Imports in different contexts
+- Memory
+- Computing analysis off the UI thread to avoid blocking
+
+## Deferred feature extraction
+e.g. offline melspectrogram demo?
+- ### Web Workers 
+
+## Real-time feature extraction
+
+- ### Audio Worklets and Essentia.j
```

Comment:
We can move the cross-browser (Firefox, Edge support) audioworklet example here, instead at the main tutorial part.

Reply:
I could, but maybe it makes more sense in context (since it is particular to AudioWorklets)?